pom.xml

```diff
@@ -42,6 +50,22 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <accumulo.version>1.8.1</accumulo.version>
     <accumulo.version.majorminor>1.8</accumulo.version.majorminor>
+    <thrift.version>0.9.3</thrift.version>
+    <zookeeper.version>3.4.6</zookeeper.version>
+    <hadoop.version>2.7.2</hadoop.version>
+    <htrace.version>3.1.0-incubating</htrace.version>
+    <slf4j.version>1.7.10</slf4j.version>
+    <json.version>chargebee-1.0</json.version>
+    <math3.version>3.6.1</math3.version>
+    <codec.version>1.7</codec.version>
+    <configuration.version>1.6</configuration.version>
+    <lang3.version>3.1</lang3.version>
+    <commons.collections.version>3.2.2</commons.collections.version>
+    <commons.lang.version>2.4</commons.lang.version>
+    <commons.logging.version>1.1.1</commons.logging.version>
+    <collections4.version>4.1</collections4.version>
+    <log4j.version>1.2.17</log4j.version>
+    <guava.version>14.0.1</guava.version>
```

Comment:
Most of these versions are used in the "copy" section below, if this is replaced by "copy-dependencies" most of these can go away.